pynopegl-utils/pynopegl_utils/export.py

```diff
@@ -199,3 +199,92 @@ def _export_worker(
 
     os.close(fd_w)
     reader.wait()
+
+
+def run():
+    import argparse
+    import re
+    from inspect import getmembers
+    from pathlib import Path
+
+    from pynopegl_utils.misc import SceneCfg
+    from pynopegl_utils.module import load_script
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        dest="input",
+        type=Path,
+        help="path to the script containing scenes",
+    )
+    parser.add_argument(
+        "-f",
+        dest="filter",
+        default=r".*",
+        help="Filter functions using a regex",
```

Comment:
Can you make the help description consistent regarding uppercasing (or not) the first letter ?

Reply:
Fixed